challenges/day5-issues/url-shortener/backend/app.py

```
@@ -0,0 +1,45 @@
+# URL Shortener (FastAPI backend)
+
+from fastapi import FastAPI, HTTPException, Request
+from pydantic import BaseModel
+from typing import Dict
+import uvicorn
+
+app = FastAPI()
+
+# In‑memory storage for demo purposes
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)